docs/howto/openstack/neutron/create-security-groups.md

```diff
@@ -0,0 +1,59 @@
+# How to Create Security Groups
+
+By definition, security groups are "[...] _sets of IP filter rules that are applied to all project instances, which define networking access to the instance. Group rules are project specific; project members can edit the default rules for their group and add new rule sets_". [[1](https://docs.openstack.org/nova/latest/admin/security-groups.html)]
+
+To create a security group click on "_Security Groups_" in the left-hand navigation menu:
+
+![navigation-panel](assets/create-security-groups/01-navigation-panel.png)
+
+and then click on "_Create new Security Group_" in the top-right corner:
+
+![create-button](assets/create-security-groups/02-create-button.png)
+
+> An alternative way to create a Security Group is by clicking on "_Create ...._" button in the top bar.
+
+Now give the security group a name and description and choose in which region to create it, then click "_create_":
+
+![create-panel](assets/create-security-groups/03-name-creation.png)
+
+## Remove default ingress rules
+
+By default, a security group named "_default_" has been already created for you, allowing any sort of traffic from any source (ingress).  
+Click on it and select the "_Rules_" tab to view its rules:
+
+![default-rules](assets/create-security-groups/04-default-rules.png)
+
+This is not recommended and a good approach is to either create and use a new security group than the default one **-or-** restrict ingress traffic to specific ports and sources.
+
+In case we want use the default group, we first need to **remove the two ingress rules** that allow all incoming traffic.
+
+![default-ingress-rules](assets/create-security-groups/05-default-ingress-rules.png)
+
+## Restrict SSH access
+
+The next thing to do, is to allow SSH access on **port 22** to the server, only from specific VPN(s). To do this we click on the "_Create new rule_" button.
+
+![create-rule-panel](assets/create-security-groups/06-create-security-group-rule.png)
+
+> If you don't know your IP, simply visit [whatsmyip.com](https://whatsmyip.com/). My IP is 91.123.199.188 so if I only want to SSH from my machine only, I insert 91.123.199.188/32 as CIDR. If I want to enable SSH access from any address in that range I will instead use 91.123.199.0/24.
```

Comment:
This is a how-to. Address the user directly, rather than referring to yourself.

Reply:
How does this sound?:
> \> If you don't know your IP, simply visit [whatsmyip.com](https://whatsmyip.com/). In this example your IP is 91.123.199.188 and you want to allow SSH from this IP address only, then enter 91.123.199.188/32 as CIDR. To allow SSH access from any address in that IP range, instead enter 91.123.199.0/24 as CIDR.

Reply:
That sounds better, and also:

* I would suggest to people that they visit https://icanhazip.com instead.
* When using "public" IP addresses in documentation, don't use real ones. Instead, use RFC5737 ones (see also #43).

Reply:
I was actually looking for reserved IPs for this use case .. Thanks, will change them. :)